src/appengine/libs/auth.py

```diff
@@ -177,6 +178,8 @@ def create_session_cookie(id_token, expires_in):
 
 def get_current_request():
   """Get the current request."""
+  if flask.request:
```

Comment:
Can we just fix up callers to access flask.request directly instead?

Reply:
do you mean to pass it as an argument to the calling function and remove this function or directly use the global request? This anyways would be changed when migration is completed.

Reply:
Directly use the global request. Unless dependencies etc make it difficult, we should just use the global request directly in the request and avoid having to migrate again later.